src/commands/editprofile/index.ts

```diff
@@ -0,0 +1,90 @@
+import type { ChatInputCommandInteraction, GuildMember } from 'discord.js';
+
+import { Command } from '../../types/command.js';
+import { formatHexColor } from '../../utilities/hex-color-helper.js';
+import type {
+  CommandBuilder,
+  CommandBuilderOutput
+} from '../../types/command-builder.js';
+import {
+  InvalidColorStringError,
+  RustyBotInvalidArgumentError
+} from '../../errors/rusty-bot-errors.js';
+import { getMemberFirestoreReference } from '../../utilities/firestore-helper.js';
+
+const MAX_ABOUT_LENGTH = 2048;
+
+export class EditProfileCommand extends Command {
```

Comment:
This is fine, but I think it might make more sense for this to be rolled as a profile subcommand. That's work I was planning to do in the future already though, so more of a TODO than a blocking change.

Reply:
Agreed. Will not make any changes now but just curious: would it be `profile view` & `profile edit`?